lib/6to5/transformation/helpers/replace-supers.js

```diff
@@ -25,6 +25,36 @@ function ReplaceSupers(methodNode, className, superName, isLoose, file) {
 }
 
 /**
+ * Sets a super class value of the named property.
+ *
+ * @example
+ *
+ *   _set(Object.getPrototypeOf(CLASS.prototype), "METHOD", "VALUE", this)
+ *
+ * @param {Node} property
+ * @param {boolean} isStatic
+ * @param {boolean} isComputed
+ *
+ * @returns {Node}
+ */
+ReplaceSupers.prototype.setSuperProperty = function (property, value, isStatic, isComputed, thisExpression) {
+  return t.callExpression(
+    this.file.addHelper("set"),
+    [
+      t.callExpression(
+        t.memberExpression(t.identifier("Object"), t.identifier("getPrototypeOf")),
+        [
+          isStatic ? this.className : t.memberExpression(this.className, t.identifier("prototype"))
+        ]
+      ),
+      isComputed ? property : t.literal(property.name),
+      t.identifier(value.name),
```

Comment:
This works fine for simple expressions with Identifiers. If the right hand side of the assignment expression is something more complex though, things go awry.